src/code_actions.zig

```diff
@@ -265,18 +259,22 @@ fn handlePointlessDiscard(builder: *Builder, actions: *std.ArrayListUnmanaged(ty
 // attempts to converts a slice of text into camelcase 'FUNCTION_NAME' -> 'functionName'
 fn createCamelcaseText(allocator: std.mem.Allocator, identifier: []const u8) ![]const u8 {
     var num_separators: usize = 0;
-    
+
+    for (identifier) |c| {
+        if (c == '_') num_separators += 1;
```

Comment:
you can refactor this into `const num_separators = std.mem.count(u8, identifier, "_")`